### PR TITLE
UI navigation

### LINKED ISSUE #69: Navigointikomponentti jolla hyppiä kysymysten välillä (nuolipainikkeet).
Käyttäjä voi muokata vastauksiaan lähetykseen asti: #34
estimaatti: 1 h

### PR DESCRIPTION
**Pull request by korolainenriikka**
nav buttons & nav button tests
button component changed: primary, secondary & submit

### CODE REVIEW

File: frontend/components/navigationButtons.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  const nextQUrl = `/survey/questions/${currentQuestionId + 1}`
+
+  const isFirstQuestion = currentQuestionId === 1
+  const isLastQuestion = currentQuestionId === surveyLength
+
+  return (
+    <NavButtonWrapper>
+      {!isFirstQuestion && 
+        <Link href={prevQUrl} passHref>
+          <Button type="secondary">
```
Comment (Nurou) [inline frontend/components/navigationButtons.jsx]: Buttons are not necessary here. Links work for navigation

File: frontend/components/navigationButtons.jsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import styled from 'styled-components'
+import Link from 'next/link'
+
+import Button from '../components/button'
+
+const NavButtonWrapper = styled.div`
```
Comment (Nurou) [inline frontend/components/navigationButtons.jsx]: HTML has an element for nav for this exact purpose:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav

File: frontend/components/navigationButtons.jsx (hunk 1 above)
Comment (Nurou) [reply frontend/components/navigationButtons.jsx]: See, e.g. https://a11y-101.com/design/button-vs-link#:~:text=A%20link%20will%20redirect%20you,like%20opening%20or%20closing%20something.

Comment (Nurou) [reply frontend/components/navigationButtons.jsx]: also, does the href need to be passed from the Link component?

Comment (korolainenriikka) [reply frontend/components/navigationButtons.jsx]: fixed?

**Review by sinisaarinen — APPROVED** [review]
OK

**Pull request merged**